modules/telco-ran-node-tuning-operator.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-ran-ref-du-components.adoc
+
+:_content-type: REFERENCE
+[id="telco-ran-node-tuning-operator_{context}"]
+= Node Tuning Operator
+
+Description::
+You tune the cluster performance by link:https://docs.openshift.com/container-platform/latest/scalability_and_performance/cnf-create-performance-profiles.html[creating a performance profile].
++
+This includes:
++
+* Selecting the realtime or non-realtime kernel.
+
+* Allocating cores to a reserved or isolated `cpuset`.
+{product-title} processes allocated to the management workload partition are pinned to reserved set.
+
+* Enabling kubelet features (CPU manager, topology manager, and memory manager)
+
+* Configuring huge pages
+
+* Setting additional kernel arguments
+
+* Setting per-core power tuning and max CPU frequency
+
+Limits and requirements::
+* Set the reserved `cpuset` to 4 or more 2-core hyperthreads on Intel 3rd Generation Xeon (Ice Lake) 2.20 GHz CPUs or better with firmware tuned for maximum performance.
+
+* Set the reserved `cpuset` to include both hyperthread siblings for each included core.
+All non-reserved cores are available as allocatable CPU for scheduling workloads.
+
+* Configure reserved and isolated CPUs to include all threads of all cores
+
+* Set Core 0 of each NUMA node to be included in the reserved CPU set
+
+* Set the huge page size to 1G
+
+* Pin OpenShift pods that are configured by default as part of the management workload partition to the reserved cores.
```

Comment:
This text gives the impression this is something the user is expcted to do, as oppsed to something which is done by the product